feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/WidgetsRepository.kt

```diff
@@ -0,0 +1,15 @@
+package cm.aptoide.pt.feature_apps.data
+
+import cm.aptoide.pt.feature_apps.domain.Widget
+import kotlinx.coroutines.flow.Flow
+
+interface WidgetsRepository {
+
+  fun getStoreWidgets(): Flow<Result>
+
+}
+
+sealed interface Result {
```

Comment:
soon we should adapt to have the generic result structure. Maybe even having aleixo's approach of having it being mapped automatically for the error cases in the lower entities.